lib/methods/import.js

```diff
@@ -59,6 +70,55 @@ module.exports = self => {
         docs: EJSON.parse(docs),
         attachments: EJSON.parse(attachments)
       };
+    },
+
+    async insertDocs(req, docs, reporting) {
+      return self.insertDocuments(
+        docs,
+        reporting,
+        doc => self.apos.modules[doc.type].insert(req, doc)
+      );
+    },
+
+    async insertAttachments(attachments, reporting) {
```

Comment:
will be good to have req as a first param

Reply:
is back